### PR TITLE
chore: add typography page

### PR DESCRIPTION
**Pull request by nkrantz**
Adds a new foundations page, typography.

### CODE REVIEW

File: packages/paste-website/src/pages/foundations/typography/index.mdx [hunk 1]
```diff
@@ -0,0 +1,692 @@
...
+<br />
+
+#### When you have lots to say, use Twilio Sans Text Regular
+
+In order to create juxtaposition and legibility within our designs, Twilio Sans Text’s Regular font weight is used for all body copy. This blends well and gives a nice balance with the headings that are Semibold.
+
+[The Paragraph component](/components/paragraph) is perfect for large blocks of content.
+
+<Card>
+  Paragraph
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/foundations/typography/index.mdx:222]: issue: technically you're not using the paragraph component here

File: packages/paste-website/src/pages/foundations/typography/index.mdx [hunk 2]
```diff
@@ -0,0 +1,692 @@
...
+<br />
+
+[Check out our line height and font size tokens.](/foundations/colors/#text-and-icon-colors)
+
+<Card>
+  <Text fontSize="fontSize80" lineHeight="lineHeight80">
+    Create a function to respond to Twilio Webhook Events like an incoming call.
+  </Text>
+  <br />
+  <InlineCode>Font-size: 28px</InlineCode>
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/foundations/typography/index.mdx:332]: question: should these be tokens or pixel sizes?

File: packages/paste-website/src/pages/foundations/typography/index.mdx [hunk 3]
```diff
@@ -0,0 +1,692 @@
...
+- [Label](/components/label)
+- [Help Text](/components/help-text)
+- [List](/components/list)
+- [Paragraph](/components/paragraph)
+- [Truncate](/components/truncate)
+
+Most of our block-level typography components like Heading, Paragraph, and Detail Text have preset margins so that stacking components will give your layout the ideal amount of spacing without having to adjust margins manually too much yourself.
+
+<LivePreview scope={{Heading, Paragraph, Card}} language="jsx" showOverflow>
+  {`<Card padding="space0">
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/foundations/typography/index.mdx:460]: question: was the lack of padding around the card content intentional?

File: packages/paste-website/src/pages/foundations/typography/index.mdx [hunk 4]
```diff
@@ -0,0 +1,692 @@
...
+
+## Let’s play with type ✨
+
+### 3-minute type zhush
+
+- Stack headlines into two or even three lines instead of keeping them one long line (44 characters or more). This helps create a compact reading unit.
+- A good rule is to use no more than 3 type styles (semibold, medium, regular fonts). This helps blend type styles properly and not get carried away.
+- Be generous with the spacing around your type. It thrives when it has room to breathe. We suggest 8–32 pixels between type if they are friends, of course 🙂. [Check out the spacing and layout guidelines for more info.](/foundations/spacing-and-layout)
+
+### Elevating your designs with type
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/foundations/typography/index.mdx:665]: thought: it definitely threw me for a second that the after came before the before.

File: packages/paste-website/src/pages/foundations/typography/index.mdx (hunk 3 above)
Comment (nkrantz) [reply packages/paste-website/src/pages/foundations/typography/index.mdx:460]: yes - to show the bottom margin difference but lmk if you think of a better way

File: packages/paste-website/src/pages/foundations/typography/index.mdx (hunk 4 above)
Comment (nkrantz) [reply packages/paste-website/src/pages/foundations/typography/index.mdx:665]: @libertycarlin

File: packages/paste-website/src/pages/foundations/typography/index.mdx (hunk 2 above)
Comment (nkrantz) [reply packages/paste-website/src/pages/foundations/typography/index.mdx:332]: @libertycarlin

File: packages/paste-website/src/pages/foundations/typography/index.mdx (hunk 3 above)
Comment (nkrantz) [reply packages/paste-website/src/pages/foundations/typography/index.mdx:458]: ([reference to example](https://docs.google.com/document/d/10EKRuDy0fO90IxYOrJ2vUm2d4ijHqXsgfqgUVWd_XyM/edit?disco=AAAArEsssHQ))

File: packages/paste-website/src/pages/foundations/typography/index.mdx (hunk 2 above)
Comment (libertycarlin) [reply packages/paste-website/src/pages/foundations/typography/index.mdx:330]: We can do tokens to match the others

**Review by libertycarlin — APPROVED** [review]
looks so good!!

**Review by SiTaggart — APPROVED** [review]

**Pull request merged**